homeassistant/components/sabnzbd/sensor.py

```diff
@@ -14,22 +14,24 @@ async def async_setup_platform(hass, config, async_add_entities, discovery_info=
     sensors = sab_api_data.sensors
     client_name = sab_api_data.name
     async_add_entities(
-        [SabnzbdSensor(sensor, sab_api_data, client_name) for sensor in sensors]
+        [
+            SabnzbdSensor(SENSOR_TYPES[sensor], sab_api_data, client_name)
+            for sensor in sensors
+        ]
     )
 
 
 class SabnzbdSensor(SensorEntity):
     """Representation of an SABnzbd sensor."""
 
-    def __init__(self, sensor_type, sabnzbd_api_data, client_name):
+    _attr_should_poll = False
+
+    def __init__(self, myDescription, sabnzbd_api_data, client_name):
```

Comment:
`myDescription` is not a valid variable name, maybe use `description` or `entity_description` instead.